lib/platform/gitlab/index.ts

```diff
@@ -1067,3 +1067,33 @@ export async function ensureCommentRemoval({
 export function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
   return Promise.resolve([]);
 }
+
+// See https://gitlab.com/gitlab-org/gitlab/-/blob/master/app/graphql/types/user_status_type.rb
+interface GitlabUserStatus {
+  message?: string;
+  message_html?: string;
+  emoji?: string;
+  availability: 'not_set' | 'busy';
+}
```

Comment:
move to `./types.ts`

Reply:
👍 e9b23f2cedbf0ca576d0bc22444694826ca19888